tests/boulder-integration.sh

```diff
@@ -50,7 +50,11 @@ dir="$root/conf/archive/le1.wtf"
 for x in cert chain fullchain privkey;
 do
     latest="$(ls -1t $dir/ | grep -e "^${x}" | head -n1)"
-    live="$(readlink -f "$root/conf/live/le1.wtf/${x}.pem")"
+    if [ `uname`  == 'Darwin' ]; then
+      live="$(greadlink -f "$root/conf/live/le1.wtf/${x}.pem")"
+    else
+      live="$(readlink -f "$root/conf/live/le1.wtf/${x}.pem")"
+    fi
```

Comment:
I often find that defining at the top of the file
```bash
if [ `uname`  == 'Darwin' ]; then
  readlink="greadlink"
else
  readlink="readlink"
fi
```

and using it later as `$readlink -f ...` nicely deduplicates, leaving less space for error and maintenance burden.